components/crud-web-apps/common/frontend/kubeflow-common-lib/projects/kubeflow/src/lib/form/name-namespace-inputs/name-input/name-input.component.html

```diff
@@ -1,9 +1,5 @@
 <mat-form-field class="wide" appearance="outline">
-  <mat-label>{{ 'common.name' | translate }}</mat-label>
-  <input
-    matInput
-    placeholder="{{ 'common.name' | translate }}"
-    [formControl]="nameControl"
-  />
-  <mat-error>{{ nameError().key | translate: nameError().params }}</mat-error>
+  <mat-label i18n>Name</mat-label>
+  <input matInput placeholder="Name" [formControl]="nameControl" />
```

Comment:
You forgot to mark the placeholder for translation here.

Reply:
Pushed a commit